npm/vite-plugin-cypress-esm/src/index.ts

```diff
@@ -209,6 +209,12 @@ export const CypressEsm = (options?: CypressEsmOptions): Plugin => {
     return code.replace(
       RE,
       (match, importVars: string, importTarget: string) => {
+        if (isImportOnIgnoreList(importTarget)) {
```

Comment:
One last commit to also ignore dynamic imports present on the `ignoreImportList`